buildings/gui/reference_data.py

```diff
@@ -15,9 +15,7 @@
 from buildings.utilities import database as db
 
 __location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
-FORM_CLASS, _ = uic.loadUiType(
-    os.path.join(os.path.dirname(__file__), "reference_data.ui")
-)
+FORM_CLASS, _ = uic.loadUiType(os.path.join(os.path.dirname(__file__), "reference_data.ui"))
```

Comment:
Can we remove the formatting changes from this PR?

We need to apply Black consistently across the whole codebase but should do that in a separate PR.